server/data.py

```diff
@@ -340,13 +340,15 @@ def _count_gt_for_write(self, variants, data, filters, chips, data_type):
                 het['three_gt_probs'] = gt_probs_het
                 hom_alt['three_gt_probs'] = gt_probs_hom
                 wt_hom['three_gt_probs'] = gt_probs_wt_hom
-                missing['three_gt_probs'] = gt_probs_missing                
+                missing['three_gt_probs'] = gt_probs_missing
+                missing['gt'] = '.|.'
+            else:
+                missing['gt'] = './.'        
```

Comment:
In future releases chip data may be phased, i.e. missing genotypes will be .|. like in imputed data - just something to keep in mind